python_modules/dagster/dagster/core/code_pointer.py

```diff
@@ -82,131 +62,87 @@ def load_python_file(python_file, working_directory):
     Takes a path to a python file and returns a loaded module
     """
     check.str_param(python_file, "python_file")
+    check.opt_str_param(working_directory, "working_directory")
```

Comment:
load_python_file and load_python_module here are the most significant parts - everything else is really just threading up working_directory